.changelog/5998.feature.md

```diff
@@ -0,0 +1 @@
+Show the progress of the history reindex in oasis control status
```

Comment:
We usually start [changelog](https://github.com/oasisprotocol/oasis-core/blob/v24.3.1/CHANGELOG.md) and git messages with a prefix, in this case `go/oasis_node/cmd` or `go/oasis_node` would be fine. 
After the title we could explain which field was added and what it shows. See previous logs for examples.